addons/portal/static/src/chatter/core/composer_patch.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Composer.fullComposer" t-inherit-mode="extension">
+        <xpath expr="//button[@title='Full composer']" position="replace"/>
+    </t>
```

Comment:
Again, can't change core mail stuff in portal.

If all of these work as we need, it means the manifest is not following proper guideline/structure, as core should always be included in all bundles.

Reply:
The portal chatter bundle is included in the frontend bundle and is completely separate from the backend bundle. So any change to the core or whatever in this bundle won't affect the backend chatter (a lot of tests would fail otherwise).